### ISSUE TITLE
Request: optional strict training / Validation of training file

### ISSUE THREAD

**Issue by karatekaneen** [kermitt2/grobid#724]
**TL;DR: It would be nice to have a way to make the process to fail if training data is corrupt**

### The problem

We are looking at automating as much as possible of the training process and it seems that it's quite simple to use the grobid batch service for almost everything. The problem we ran into are if we have mistakenly corrupted the training data (as I expertly did in #659 ) where the training would continue even if the file is corrupt.

### Proposed solutions

One possible solution would be to add a "strict" flag when training that removes the tolerance for existing corrupt files and exits with an error status that is possible to handle in other applications (non-Java).
Another solution that might require some more work would be that add a validation feature that checks if the training data is valid.
